src/accelerate/accelerator.py

```diff
@@ -437,9 +438,9 @@ def __init__(
         self.has_fp8_handler = False
         if kwargs_handlers is not None:
             for handler in kwargs_handlers:
-                assert isinstance(handler, KwargsHandler), (
-                    f"Unsupported kwargs handler passed: {handler}, must be one that inherits `accelerate.utils.KwargsHandler`."
-                )
+                assert isinstance(
+                    handler, KwargsHandler
+                ), f"Unsupported kwargs handler passed: {handler}, must be one that inherits `accelerate.utils.KwargsHandler`."
```

Comment:
Not sure why this is getting hit by the ruff formatter; I can revert and commit with `--no-verify` if we don't want the formatting change here